docs/commerce/cart/cart.md

```diff
@@ -0,0 +1,100 @@
+---
+description: The Cart component covers adding items to the shopping cart, as well as previewing and modifying the cart information.
+edition: commerce
+---
+
+# Cart
+
+The Cart component is a foundation of the Commerce offering delivered as part 
+of [[= product_name =]].
+It covers actions related to the creation and handling of a list of products 
+that the buyer intends to purchase.
+
+
+The Cart component exposes the following:
+
+- [PHP API](cart_api.md) that allows for managing carts and cart entries, or validating products
```

Comment:
"validating products" is too vague in my opinion. The API indeed has `validateCart` method but it does much more than just validating products.